guides/ssl-certificates/cloudflare.md

```diff
@@ -95,13 +87,13 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: cloudflare-api-token-secret
-  namespace: coder # Your Coder deployment namespace
+  namespace: cert-manager # Your cert-manager deployment namespace
```

Comment:
I was positive the secret worked well when in `coder`, but wanted to sanity check that it would break if changed to `cert-manager`. It does. Recorded a quick video: https://youtu.be/i1fSP32MmPU

basically:
- if person is using ClusterIssuer, secret needs to be in `cert-manager` namespace
- if person is using Issuer, secret needs to be in sane namespace as Issuer

Reply:
I also personally like recommending an Issuer over a ClusterIssuer, so I just suggest reverting this change. However, it seems like all the other guides are `ClusterIssuer` first. If you do that, then we should reverse the lower section comparing the two.

Reply:
@mterhar Do you have any preferences on recommending Issuer over ClusterIssuer? I'd be happy to make the edits but I'm not sure which route works best for users

Reply:
Looks like Ben has discovered a nuance that I was missing.  It's tough to know which will be the least friction for users.  It may make sense to have all of our specs reference "issuer" and set the namespaces for issuer and secrets to "coder" for internal consistency and then reference the clusterissuer secret namespace in a separate `TLS troubleshooting` section.

Reply:
@deansheather is also quite well-versed in cert-manager shenanigans, maybe worth getting a second opinion from him

Reply:
```suggestion
  namespace: coder # Your coder deployment namespace
```

I am suggesting this change so we can just merge this doc.